src/libraries/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -4,6 +4,8 @@
 // Changes to this file must follow the https://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System.Diagnostics.CodeAnalysis;
```

Comment:
`ref` files are generated and use full qualified names in source. You shouldn't hand edit them because the next time someone wants to update it, they need to manually revert your changes.

Reply:
See [https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md) for how to update ref files.

---
In reply to: [595286512](https://github.com/dotnet/runtime/pull/49682#discussion_r595286512) [](ancestors = 595286512)